src/main/java/com/jvm_bloggers/frontend/public_area/all_issues/AllIssuesPageBackingBean.java

```diff
@@ -33,7 +34,11 @@ public AllIssuesPageBackingBean(NewsletterIssueForListingQuery query) {
                     .sorted(Comparator.comparing(NewsletterIssueForListing::getIssueNumber))
                     .reverse()
             );
-        return TreeMap.ofAll(issuesGroupedByYearMonth.toJavaMap());
```

Comment:
Factory method by default use `EntryComparator.natural()` to sort keys - this is way months were is ASC order.